x-pack/plugins/cross_cluster_replication/public/app/services/routing.js

```diff
@@ -15,6 +15,16 @@ const isModifiedEvent = event => !!(event.metaKey || event.altKey || event.ctrlK
 
 const isLeftClickEvent = event => event.button === 0;
 
+const queryParamsFromObject = params => (
```

Comment:
We can use `stringify` method of the [`querystring` package](https://nodejs.org/api/querystring.html) here instead of hand-rolling this logic. 

```js
import { stringify } from 'querystring';

const paramsStr = stringify(params, '&', '=', {
  encodeURIComponent: (val) => val, // Don't encode special chars
});
return `?${paramsStr}`;
```

Reply:
Great, yes much better.